compiler/common-artifacts/exclude.lst

```diff
@@ -5,6 +5,7 @@
 
 #[[ optimize : Exclude from circle optimization(circle2circle) ]]
 ## TensorFlowLiteRecipes
+optimize(CumSum_000)
```

Comment:
Just for note, the build will work without this line because there's no optimization pass related `CumSum` that would make it failure.

Reply:
Thanks for comment.
By the way, while it tests with execution of circle2circle, it complained about no implementation in luci/export regarding to CumSum. And it breaks the build. 
So, the line will be removed after luci/export commit is merged.

Reply:
Ah, right. Sorry. I was mistaken.